templates/_lib.hcl

```diff
@@ -85,6 +85,8 @@ ephemeral_disk {
         }
       }
     }
+
+    ${ promtail() }
```

Comment:
Maybe rename this to `promtail_task` to make it clear what's being generated?

Reply:
Fair enough.